var/spack/repos/builtin/packages/dyninst/package.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```

Comment:
This is a ginormous update in this file :)

Reply:
synced with upstream again and you see more updates